apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/CompilationUnit.kt

```diff
@@ -0,0 +1,148 @@
+package com.apollographql.apollo.gradle
```

Comment:
nit: seems like the indents in this file is 4 spaces instead of 2?

Reply:
Oh my, sorry about that. Could we merge that PR: https://github.com/apollographql/apollo-android/pull/1326 ?